openmetadata-ui/src/main/resources/ui/playwright/support/team/TeamClass.ts

```diff
@@ -45,9 +45,12 @@ export class TeamClass {
     return this.responseData;
   }
 
-  async create(apiContext: APIRequestContext) {
+  async create(
+    apiContext: APIRequestContext,
+    extraData?: Record<string, unknown>
```

Comment:
Why do we need to introduce extra data. You can pass your data while object creation